opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -0,0 +1,96 @@
+use crate::transform::transform_resource_metrics;
+use async_trait::async_trait;
+
+use opentelemetry_api::metrics::{MetricsError, Result};
+use opentelemetry_sdk::metrics::{
+    data::{ResourceMetrics, Temporality},
+    exporter::PushMetricsExporter,
+    reader::{AggregationSelector, TemporalitySelector},
+    Aggregation, InstrumentKind,
+};
+
+use crate::tracepoint;
+use prost::Message;
+use std::fmt::{Debug, Formatter};
+
+#[derive(Clone, Copy)]
+pub struct MetricsExporter {}
+
+impl MetricsExporter {
+    pub fn new() -> MetricsExporter {
+        // This is unsafe because if the code is used in a shared object (DLL),
+        // the event MUST be unregistered before the shared object unloads.
+        unsafe {
+            let result = tracepoint::register();
+            if result != 0 {
+                println!("Tracepoint failed to register.");
```

Comment:
nit -  As discussed in the community meeting today, we should eventually use the custom error handler here. If not planned for this PR, good to add TODO here. 

Also if we plan to add TODO here, replace the `println!` with `eprintln!`, to ensure that the error messages are written to stderr, not the stdout. This holds true for other places in this PR.

Reply:
Fixed.